core/src/main/java/org/apache/iceberg/avro/Avro.java

```diff
@@ -170,6 +171,7 @@ public static ReadBuilder read(InputFile file) {
     private final ClassLoader defaultLoader = Thread.currentThread().getContextClassLoader();
     private final InputFile file;
     private final Map<String, String> renames = Maps.newLinkedHashMap();
+    private NameMapping nameMapping;
```

Comment:
Instance fields should be defaulted. Can you set this to `null`?

Reply:
Is this for code readability, because AFAIK it is redundant for Java?